packages/flutter_tools/lib/src/html_utils.dart

```diff
@@ -0,0 +1,112 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:html/dom.dart';
+import 'package:html/parser.dart';
```

Comment:
Is this maybe causing the internal error

```
(13:25:14) ERROR: third_party/dart/flutter_tools/BUILD:26:13: in _dart_library rule //third_party/dart/flutter_tools:api: 
target '//third_party/dart/html:html' is not visible from target '//third_party/dart/flutter_tools:api'. 
Check the visibility declaration of the former target if you think the dependency is legitimate
```

Reply:
Yep, I'm working on a g3fix already :)